tldw_Server_API/tests/Chat/integration/conftest.py

```
@@ -0,0 +1,82 @@
+            cfg = _llc()
+            if "openai_api" not in cfg:
+                cfg["openai_api"] = {}
+            cfg["openai_api"]["api_key"] = os.environ["OPENAI_API_KEY"]
+            cfg["openai_api"]["api_base_url"] = f"{mock_openai_server}/v1"
+            # Replace loader functions to return patched cfg
+            import tldw_Server_API.app.core.config as _config_mod
+            _config_mod.load_and_log_configs = lambda: cfg
+            import tldw_Server_API.app.core.LLM_Calls.LLM_API_Calls as _llm_calls_mod
+            _llm_calls_mod.load_and_log_configs = _config_mod.load_and_log_configs
+        except Exception:
+            pass
```

Ruff (0.14.4)

`try`-`except`-`pass` detected, consider logging the exception

(S110)